base/applications/games/winmine/rsrc.rc

```diff
@@ -51,6 +51,9 @@ IDI_MINES BITMAP "rc/mines.bmp"
 /* UTF-8 */
 #pragma code_page(65001)
 
+#ifdef LANGUAGE_AR_AR
+    #include "lang/ar-AR.rc"
+#endif
```

Comment:
There's no ar-AR, check existing Arabic locales here: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/cmake/localization.cmake;hb=17b0623cf8428fa547e2c81d088bac1e0ceb96d1#l9

Reply:
There is no ar-001 , So I might change it to ar-sa instead